biocrnpyler/dna_assembly_promoter.py

```diff
@@ -108,11 +108,13 @@ def update_reactions(self):
 
 #A class for a promoter which can be activated by a single species, modelled as a positive hill function
 class ActivatablePromotor(Promoter):
-    def __init__(self, name, activator, transcript = None, **keywords):
+    def __init__(self, name, activator, leak = False, transcript = None, **keywords):
```

Comment:
This is fine for now, but this change breaks all the code which use the old format with positional arguments:
```python
ap = ActivatablePromotor(name,activator,transcript)
```
I guess this class is not used extensively right now, so it's fine for now

Reply:
I will fix this too - good catch thanks!